app/src/main/java/mozilla/lockbox/action/RouteAction.kt

```diff
@@ -46,7 +46,7 @@ sealed class RouteAction(
     sealed class AppWebPage(
         val url: String? = null,
         @StringRes val title: Int? = null
-    ) : RouteAction() {
+    ) : RouteAction(TelemetryEventMethod.tap, TelemetryEventObject.app_web_page) {
```

Comment:
The [telemetry](https://github.com/mozilla-lockbox/lockbox-android/blob/master/docs/metrics.md#list-of-proposed-events) docs calls for these to be more specific (e.g., "show faq" vs "opened web app page").

Instead, consider something like:

```
seald class AppWebPage(
  val url: String? = null,
  @StringRes val title: Int? = null,
  eventObject: TelemetryEventObject
) : RouteAction(TelemetryEventMethod.show, eventObject) {
  object FaqList  : AppWebPage(Constant.Faq.uri, R.string.nav_menu_faq, TelemetryEventMethod.settings_faq)
  object SendFeedback : AppWebPage(Constant.SendFeedback.uri, R.string.nav_menu_feedback, TelemetryEventObject.settings_provide_feedback)
}
```

Reply:
awesome! thank you!